src/btree/row_key.c

```diff
@@ -270,61 +270,97 @@ __wt_row_leaf_key_work(
              */
             if (slot_offset == 0) {
                 __wt_readlock(session, &btree->ovfl_lock);
-                copy = WT_ROW_KEY_COPY(rip);
-                if (!__wt_row_leaf_key_info(page, copy, NULL, &cell, &keyb->data, &keyb->size)) {
-                    __wt_cell_unpack_kv(session, page->dsk, cell, unpack);
-                    ret = __wt_dsk_cell_data_ref(session, WT_PAGE_ROW_LEAF, unpack, keyb);
-                }
+                /*
+                 * KEITH I think there's a race here -- it's a long one, but if the object is
```

Comment:
Same here.

Reply:
This isn't a race I introduced, and it's non-trivial to fix. I removed my name in c4d658436 and I'll open a JIRA ticket.